---a/net/minecraft/world/biome/BiomeGenDesert.java
+++b/net/minecraft/world/biome/BiomeGenDesert.java
@@ -1,15 +1,17 @@
 package net.minecraft.world.biome;
 
 import java.util.Random;
+
 import net.minecraft.init.Blocks;
 import net.minecraft.util.BlockPos;
 import net.minecraft.world.World;
 import net.minecraft.world.gen.feature.WorldGenDesertWells;
 
-public class BiomeGenDesert extends BiomeGenBase
-{
+public class BiomeGenDesert extends BiomeGenBase {
+
     private static final String __OBFID = "CL_00000167";
 
-    public BiomeGenDesert(int p_i1977_1_)
-    {
+    public BiomeGenDesert(int p_i1977_1_) {
         super(p_i1977_1_);
         this.spawnableCreatureList.clear();
         this.topBlock = Blocks.sand.getDefaultState();
@@ -23,17 +23,14 @@
         this.spawnableCreatureList.clear();
     }
 
-    public void decorate(World worldIn, Random p_180624_2_, BlockPos p_180624_3_)
-    {
+    public void decorate(World worldIn, Random p_180624_2_, BlockPos p_180624_3_) {
         super.decorate(worldIn, p_180624_2_, p_180624_3_);
 
-        if (p_180624_2_.nextInt(1000) == 0)
-        {
+        if (p_180624_2_.nextInt(1000) == 0) {
             int i = p_180624_2_.nextInt(16) + 8;
             int j = p_180624_2_.nextInt(16) + 8;
             BlockPos blockpos1 = worldIn.getHorizon(p_180624_3_.add(i, 0, j)).up();
             (new WorldGenDesertWells()).generate(worldIn, p_180624_2_, blockpos1);
         }
     }
-}
+}
